application/client/src/app/ui/views/workspace/headers/template.html

```diff
@@ -1,10 +1,17 @@
 <span class="offset" [ngStyle]="ngGetOffsetStyle()"></span>
-<span class="header" 
+<span class="header"
       *ngFor="let header of headers; let i = index"
```

Comment:
looks like we need `uuid` of each column to correctly bind it between view and controller or static `index` (as property in header description)